roles/openshift_storage_nfs_lvm/defaults/main.yml

```diff
@@ -8,3 +8,10 @@ osnl_mount_dir: /exports/openshift
 
 # Volume Group to use.
 osnl_volume_group: openshiftvg
+
+# Volume reclaim policy of a PersistentVolume tells the cluster
+# what to do with the volume after it is released.
+#
+# Valid values are "Retain" or "Recycle".
+# See https://docs.openshift.com/enterprise/3.0/architecture/additional_concepts/storage.html#pv-recycling-policy
+osnl_volume_reclaim_policy: "Recycle"
```

Comment:
Should be OK now, added a newline and rebased+pushed. `yamllint` also complains about the link on line 16 (too long), should I remove it?

```bash
→ yamllint roles/openshift_storage_nfs_lvm/defaults/main.yml
roles/openshift_storage_nfs_lvm/defaults/main.yml
  16:81     error    line too long (113 > 80 characters)  (line-length)
```